src/js/BookReader.js

```diff
@@ -1789,15 +1794,17 @@ BookReader.prototype.prefetchImg = function(index) {
   }
 
   if (loadImage) {
-    var img = document.createElement("img");
-    $(img).addClass('BRpageimage').addClass('BRnoselect');
+    const div = this.leafTemplate({}, index);
+    $('<img />', {
+      'class': 'BRpage-frame BRpageimage',
+      src: pageURI
+    }).appendTo(div);
     if (index < 0 || index > (this._models.book.getNumLeafs() - 1) ) {
       // Facing page at beginning or end, or beyond
-      $(img).addClass('BRemptypage');
+      div.addClass('BRemptypage');
     }
-    img.src = pageURI;
-    img.uri = pageURI; // browser may rewrite src so we stash raw URI here
-    this.prefetchedImgs[index] = img;
+    div[0].uri = pageURI;
```

Comment:
Let's keep the comment. I can't find anywhere that uses this `uri` field, but let's keep the comment for now.
```suggestion
    div[0].uri = pageURI; // browser may rewrite src so we stash raw URI here
```